packages/@okta/i18n/src/properties/login.properties

```diff
@@ -746,7 +746,6 @@ oie.password.authenticator.description = Choose a password for your account
 oie.password.challenge.title = Verify with your password
 oie.password.enroll.title = Set up password
 oie.password.passwordLabel = Enter password
```

Comment:
What is our plan for these key/values that also exist in v1 but are modified for OIE?

Reply:
Don't have a good answer right now, as we need to involve UX/PM to revisit some of these that were modified for OIE flows. I would think a migration guide for customers with customization would be helpful